lib/Monitor/RunFolder/Staging.pm

```diff
@@ -275,8 +331,20 @@ sub _change_group {
 
     my $gid = getgrnam($group);
     chown -1, $gid, $directory;
+    # If needed, set group sticky bit
```

Comment:
Terminology is non-obvious and I've got this wrong in the past. We're not using the "sticky" bit:
S_ISGID: take a new file's group from parent directory,
S_ISVTX: sticky bit (restricted deletion flag)